README.md

```diff
@@ -17,18 +17,37 @@ These build rules are used for building [Sass][sass] projects with Bazel.
 
 <a name="setup"></a>
 ## Setup
-To  use the Sass rules, add the following to your `WORKSPACE` file to add the
-external repositories for Sass:
+These rules currently support multiple sass compilers. See documentation in
+sass/sass.bzl.
 
+If you use the pure-JavaScript option (the default) then first add this to your
```

Comment:
Should these be in the opposite order? I.e., first add rules_sass, then if you're using JS add this other thing.